src/SafeGUI.java

```diff
@@ -81,20 +81,23 @@ public void start(Stage primaryStage) {
         primaryStage.setScene(scene);
         primaryStage.show();
 
-
         try {
```

Comment:
This is causing the program to stay running even after the safe window is closed, can you fix this?